lib/eclipse/Deck/DeckKeyword.cpp

```diff
@@ -23,19 +23,19 @@
 
 namespace Opm {
 
-    DeckKeyword::DeckKeyword(const std::string& keywordName) {
+    DeckKeyword::DeckKeyword(const std::string& keywordName) :
+        m_keywordName(keywordName)
+    {
         m_knownKeyword = true;
-        m_keywordName = keywordName;
         m_isDataKeyword = false;
-        m_fileName = "";
         m_lineNumber = -1;
     }
 
-    DeckKeyword::DeckKeyword(const std::string& keywordName, bool knownKeyword) {
+    DeckKeyword::DeckKeyword(const std::string& keywordName, bool knownKeyword) :
+        m_keywordName(keywordName)
+    {
         m_knownKeyword = knownKeyword;
```

Comment:
Yes though it doesnt matter for primitives (no default constructors to call). If you prefer to use init list you shall have init list.